bindata/manifests/daemon/daemonset.yaml

```diff
@@ -74,6 +74,8 @@ spec:
                 fieldPath: metadata.namespace
           - name: CLUSTER_TYPE
             value: "{{.ClusterType}}"
+          - name: DEV_MODE
+            value: "{{.DevMode}}"
```

Comment:
idem

Reply:
setting to resolved per:
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/252#issuecomment-1066676597